package.json

```diff
@@ -209,7 +211,7 @@
 		"zod": "^3.20.2"
 	},
 	"overrides": {
-		"next": "$next",
+		"next": "^14.2.1",
```

Comment:
Can you share what this change is for?

Reply:
Ah, good catch! This was meant to allow `dev-portal` to run locally without issue. I have been getting an error stating that Next didn't understand what `$next` was referencing, I believe 🤔

The issue seems to be resolved after a downgraded my Node version, so I'll revert this :+1:

Reply:
Addressed here:
3bd0896